<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D3</class>
 <widget class="QWidget" name="D3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnIMPPant">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>632</y>
     <width>134</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 208, 200);
selection-background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>IMP PANT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btMicroNet">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>110</y>
     <width>108</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>MicroNet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod42">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>297</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NODE42</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod41">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>297</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NODE41</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbUnit3">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>647</y>
     <width>190</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(175, 175, 175);</string>
   </property>
   <property name="text">
    <string>    UNIT3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>13</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>662</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>662</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/Resources/PNG/D3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod18">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>297</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NODE18</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod39">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>297</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NODE39</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod40">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>297</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NODE40</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>106</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RED DE CÃ“MPUTO 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod42_2">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>149</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>LNET
D1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod18_2">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>149</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>LNET
A1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod40_2">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>149</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>LNET
D1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod39_2">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>149</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>LNET
D1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btNod41_2">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>149</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>LNET
D1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>151</width>
     <height>179</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>100</number>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btind1_3">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>140</y>
     <width>106</width>
     <height>182</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/Resources/PNG/Extras/transparente.svg</normaloff>:/Resources/PNG/Extras/transparente.svg</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btnind1_4">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>140</y>
     <width>106</width>
     <height>182</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/Resources/PNG/Extras/transparente.svg</normaloff>:/Resources/PNG/Extras/transparente.svg</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btind1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>106</width>
     <height>182</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/Resources/PNG/Extras/transparente.svg</normaloff>:/Resources/PNG/Extras/transparente.svg</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btnind1_5">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>140</y>
     <width>106</width>
     <height>182</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/Resources/PNG/Extras/transparente.svg</normaloff>:/Resources/PNG/Extras/transparente.svg</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btind1_2">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>140</y>
     <width>106</width>
     <height>182</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/Resources/PNG/Extras/transparente.svg</normaloff>:/Resources/PNG/Extras/transparente.svg</iconset>
   </property>
  </widget>
  <zorder>lbBack</zorder>
  <zorder>btnIMPPant</zorder>
  <zorder>btMicroNet</zorder>
  <zorder>btNod42</zorder>
  <zorder>btNod41</zorder>
  <zorder>lbUnit3</zorder>
  <zorder>btNod18</zorder>
  <zorder>btNod39</zorder>
  <zorder>btNod40</zorder>
  <zorder>label_2</zorder>
  <zorder>btNod42_2</zorder>
  <zorder>btNod18_2</zorder>
  <zorder>btNod40_2</zorder>
  <zorder>btNod39_2</zorder>
  <zorder>btNod41_2</zorder>
  <zorder>btind1_2</zorder>
  <zorder>btnind1_5</zorder>
  <zorder>btind1</zorder>
  <zorder>btnind1_4</zorder>
  <zorder>btind1_3</zorder>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
